support/noop.js

```diff
@@ -0,0 +1,2 @@
+
+module.exports = function() { return function () {} };
```

Comment:
ESLint complains when running `gulp test`:
```
support/noop.js
  2:26  error  Missing space before function parentheses  space-before-function-paren
  2:52  error  Missing semicolon                          semi
```

This is causing the Travis build to completely fail.